apps/dashboard/src/app/(dashboard)/dashboard/connect/ecosystem/[slug]/(active)/hooks/use-ecosystem.ts

```diff
@@ -8,15 +8,20 @@ export function useEcosystem({
   slug,
   refetchInterval,
   refetchOnWindowFocus,
-}: { slug: string; refetchInterval?: number; refetchOnWindowFocus?: boolean }) {
+  initialData,
+}: {
+  slug: string;
+  refetchInterval?: number;
+  refetchOnWindowFocus?: boolean;
+  initialData?: Ecosystem;
+}) {
   const { isLoggedIn } = useLoggedInUser();
 
   const ecosystemQuery = useQuery({
     queryKey: ["ecosystems", slug],
     queryFn: async () => {
       const res = await fetch(
         `${THIRDWEB_API_HOST}/v1/ecosystem-wallet/${slug}`,
-        { credentials: "include" },
```

Comment:
No auth needed here?

Reply:
I see a fetchEcosystem function below that hits the same endpoint, with the auth header. Prob want to reuse that here